libs/cart/guides/state.md

```diff
@@ -82,5 +83,56 @@ export class CartItemComponent implements OnInit {
 
 > In this example, three observable streams are assigned from `cartFacade`. Then when `addSimpleItem` is called, the `cartFacade`'s  `dispatch` function is called with the appropriately formed input. The input data is then sent off to the backend and the three observable streams are updated when a response is received.
 
+## Accessing State with Selectors
+
+Accessing state with the facade is recommended but for greater flexibility the redux store's state can be directly accessed with Daffodil's selectors.
+
+To properly maintain memoization of selectors, Daffodil selectors are hidden behind an exported getter function. This function returns an object that contains the selectors.
```

Comment:
"To properly maintain memoization of selectors that are compatible with generic models" to give the full reason why.

Reply:
changed